src/components/content.js

```diff
@@ -698,15 +699,25 @@ class Content extends React.Component {
     if (this.props.readOnly) return
     if (!this.isInEditor(event.target)) return
 
-    event.preventDefault()
+    const handleData = (data) => {
```

Comment:
It seems like we need the behavior to be synchronous, otherwise we don't have the opportunity to change the pasted content—either to prevent the paste, or to paste using the internal data model.

Is there any way for IE to be fixed without letting the paste occur already?